app.js

```diff
@@ -0,0 +1,44 @@
+const navigationSlideIn = () => {
+
+    console.log("CALL")
+    const navLinks = document.querySelector('.nav__wrapper');

```

Comment:
Noi in general folosim selectori separati pentru JS care incep cu `.js-blabla` in felul asta daca se modifica cod-ul sau css-ul nu se strica functionalitatea JS.